README.md

```diff
@@ -129,7 +135,18 @@ Let's say the Posts in the previous question are available at `http://localhost:
     3. If Step 1 is unsuccessful, print an error message to the console  
 
 ```js
-// Your answer goes here...
+$(document).ready(()=>{
+    var url = "http://localhost:3000/posts.json"
+      $.ajax({
+        url: url,
+        type: "get",
+        dataType: "json"
+      }).done(() => {
+        console.log("@posts")
```

Comment:
This will print the string `"@posts"`. How do we go about actually printing out the response?